pkg/env/env.go

```diff
@@ -77,7 +77,12 @@ func CreatePFlags(set *pflag.FlagSet, value interface{}) {
 
 	for i, parameter := range parameters {
 		if set.Lookup(parameter.Name) == nil {
-			set.Var(&flag{value: parameter.DefaultValue}, parameter.Name, descriptions[i])
+			switch val := parameter.DefaultValue.(type) {
+			case []string:
```

Comment:
remove this and add the following to the `flag.Type` method:
```golang
if kind == reflect.Slice {
	return fmt.Sprintf("%sSlice", typeOfValue.Elem().Name())
}
```

Reply:
This opens the possibility of calling .Set() on the flag, which won't reflect the way a stringSlice flag would behave, could cause issues in the future. What do you think?